packages/flutter/test/material/app_bar_test.dart

```diff
@@ -844,6 +845,45 @@ void main() {
     expect(appBarBottom(tester), kTextTabBarHeight);
   });
 
+  testWidgets('SliverAppBar expandedHeight, collapsedHeight', (WidgetTester tester) async {
+    await tester.pumpWidget(buildSliverAppBarApp(
+      floating: false,
+      pinned: false,
+      collapsedHeight: 200.0,
+      expandedHeight: 400.0,
+    ));
+
+    final ScrollController controller = primaryScrollController(tester);
+    expect(controller.offset, 0.0);
+    expect(find.byType(SliverAppBar), findsOneWidget);
+    expect(appBarHeight(tester), 400.0);
+
+    const double initialAppBarHeight = 400.0;
```

Comment:
Maybe these consts should be moved to the top of the test and then used above to not duplicate?

Reply:
Done!